components/blocks/button.js

```diff
@@ -2,12 +2,20 @@ import Link from "next/link";
 
 import styles from "./button.module.css";
 
-const Button = ({ children, link }) => {
-  return (
-    <Link href={link}>
-      <button className={styles.Button}>{children}</button>
-    </Link>
-  );
+const Button = ({ children, link, target }) => {
+  if (target) {
+    return (
+      <a href={link} target={target}>
+        <button className={styles.Button}>{children}</button>
+      </a>
+    );
+  } else {
```

Comment:
Feel free to ignore if you feel it hurts with the legibility of the code, but the `if...else` statement could be simplified by using a ternary operator, like so:

```
target ? <a href={link} target={target} className={styles.Button}>
    {children}
</a> : <Link href={link}>
     <a className={styles.Button}>{children}</a>
</Link>
```

Reply:
Doesn't hurt legibility much, so fixed on 0d1f981.